modules/distribution/src/repository/resources/conf/key-mappings.json

```diff
@@ -0,0 +1,82 @@
+{
```

Comment:
Remove separation new lines. JSON we don't keep new lines.

Reply:
We added the new lines to group the configs, so it will be more readable for a developer.

Reply:
Let's have a discussion and sort it out separately. So this is OK for the PR as it will not harm the functionality.